traffic_ops/traffic_ops_golang/server/servers.go

```diff
@@ -611,7 +611,8 @@ JOIN cdn cdn ON s.cdn_id = cdn.id
 JOIN phys_location pl ON s.phys_location = pl.id
 JOIN profile p ON s.profile = p.id
 JOIN status st ON s.status = st.id
-JOIN type t ON s.type = t.id ` + where +
+JOIN type t ON s.type = t.id 
+FULL OUTER JOIN deliveryservice_server dss ON dss.server = s.id ` + where +
```

Comment:
As with the comment above, we don't always need this join, only when dsId is a query parameter.

Reply:
Fixed